v3/src/components/graph/models/axis-model.ts

```diff
@@ -1,6 +1,6 @@
 import {Instance, types} from "mobx-state-tree"
 
-export const AxisPlaces = ["bottom", "left", "right", "top"] as const
+export const AxisPlaces = ["bottom", "left", "right", "top", "caption"] as const
```

Comment:
We can talk about this more when we discuss the DataConfiguration, but in my current formulation `caption` is a `GraphAttrPlace` rather than an `AxisPlace`.